frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,91 @@
+import { useNavigate } from 'react-router-dom';
+import { getAuth, signInWithEmailAndPassword } from "firebase/auth";
+import React from 'react';
+import PageTemplate from '../components/PageTemplate';
+
+function Login () {
+  const [email, setEmail] = React.useState('');
+  const [password, setPassword] = React.useState('');
+  const navigate = useNavigate();
+  
+  const navToRegister = () => {
+    navigate('/admin/auth/register');
+  }
+
+  const login = async (email: string, password: string) => {
+    try {
+      await signInWithEmailAndPassword(auth, email, password);
+      console.log('logged');
+    } catch (e) {
+      alert(e);
+    }
+  };
+
+  const auth = getAuth();
```

Comment:
import `auth` from `firebase.ts` , so that we only have one reference to the auth object